packages/fluentui/react-northstar-fela-renderer/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@fluentui/react-northstar-fela-renderer",
+  "description": "A CSS-in-JS renderer based on Fela for FluentUI React Northstar.",
+  "version": "0.49.0",
+  "author": "Oleksandr Fediashov <a@fedyashov.com>",
```

Comment:
I don't think we are adding `author`

Reply:
It was copied from another package, removing it in this package